profiler/src/ProfilerEngine/Datadog.Profiler.Native/CorProfilerCallback.cpp

```diff
@@ -784,10 +829,14 @@ HRESULT STDMETHODCALLTYPE CorProfilerCallback::Initialize(IUnknown* corProfilerI
         // Listen to interesting events:
         //  - AllocationTick_V4
         //  - ContentionStop_V1
+        //  - GC related events
 
         UINT64 activatedKeywords = 0;
 
-        if (_pConfiguration->IsAllocationProfilingEnabled())
+        if (
+            (_pConfiguration->IsAllocationProfilingEnabled()) ||
+            (_pConfiguration->IsGarbageCollectionProfilingEnabled())
+            )
```

Comment:
put the comment here as I cannot add the comment on line 853:
Should we play with the verbosity level ?

For example allocation profiler needs the verbosity 5, but contention or GC profiler might not need this level.
We would choose the highest one depending on the enabled profilers.

Reply:
✅ Good idea (done)